docs/administration-guide/01-managing-databases.md

```diff
@@ -57,26 +57,39 @@ To add a database, you'll need its connection information.
 
 Metabase automatically tries to connect to databases with and without SSL. If it is possible to connect to your database with a SSL connection, Metabase will make that the default setting for your database. You can always change this setting later if you prefer to connect without this layer of security, but we highly recommend keeping SSL turned on to keep your data secure.
 
-### Database Analysis
+### Database Sync and Analysis
 
-When Metabase connects to your database, it tries to decipher the field types in your tables based on each field's name. Metabase also takes a sample of each table to look for URL's, json, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Metadata** tab in the Admin Panel.
+By default, Metabase performs a lightweight hourly sync of your database, and a nightly deeper analysis of the fields in your tables to power some of Metabase's features, like filter widgets.
 
-### Metadata Syncing
+If you'd like to change these default settings, find on click on your database in the Databases section of the Admin Panel, and turn on the toggle at the bottom of the form that says "This is a large database, so let me choose when Metabase syncs and scans." (This is an option that used to be called "Enable in-depth analysis.")
```

Comment:
I'm assuming `find on click` is meant to be `find and click`?

Reply:
Yep, thanks